pkg/build/api/util.go

```diff
@@ -24,6 +24,15 @@ func StrategyType(strategy BuildStrategy) string {
 	return ""
 }
 
+// SchedulePolicyType returns string representation of given
+// BuildSchedulePolicy.
+func SchedulePolicyType(policy *BuildSchedulePolicy) string {
+	if policy == nil {
+		return "serial"
```

Comment:
This is not where we should be doing defaulting - that should always be in the API.